rocksdb_admin/admin_handler.cpp

```diff
@@ -437,9 +437,17 @@ void deleteTmpDBs() {
 
 namespace admin {
 
+AdminHandler::AdminHandler(
+  std::unique_ptr<ApplicationDBManager> db_manager,
+  RocksDBOptionsGeneratorType rocksdb_options): AdminHandler(
```

Comment:
This is the old constructor signature it will just wrap the old function type and pass it as new function type.